src/ProductDetail.jsx

```diff
@@ -4,29 +4,28 @@ import "./OneEntry.css"
 import { Box } from "./components/Box";
 import { Text } from "./components/Text";
 
-const OneEntry = () =>{
-    const id = useParams()
+const ProductDetail = () =>{
```

Comment:
You should have a loading state on this page as well. It will get rid of some of the awkwardness of seeing this state. Instead, it would be replaces by a spinner or some type of loading indicator.

<img width="1146" alt="Screenshot 2023-05-07 at 10 20 34 AM" src="https://user-images.githubusercontent.com/46625943/236683222-0c4ad446-46ed-41c6-831f-993df1e03273.png">